tests/rdma/run.py

```
@@ -14,16 +14,20 @@
 import netifaces
 import time
 import argparse
+import sys
+import signal
 
 def build_program():
     valkeydir = os.path.dirname(os.path.abspath(__file__)) + "/../.."
     cmd = "make -C " + valkeydir + "/tests/rdma"
-    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Ruff (0.15.18)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)